crates/evm_mlir/.gitignore

```diff
@@ -0,0 +1,12 @@
+**/target
```

Comment:
remove this

Reply:
Removed the evm_mlir gitignore here [c800138](https://github.com/lambdaclass/ethereum_rust/pull/548/commits/c8001387fe182aaf26a52a1565ef772ae0a7f81c)